src/main/java/edu/cornell/kfs/gl/CuGeneralLedgerConstants.java

```diff
@@ -17,9 +17,9 @@ public static class OrganizationReversionProcess {
 		public static final String ORGANIZATION_REVERSION_COA = "ORGANIZATION_REVERSION_COA";
```

Comment:
Is this constant no longer used? Can it be removed as part of the upgrade or is that out of scope?